how-it-works/2_chainkeytech/04_chain_key_tokens.card.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Chain-key tokens
+abstract: This feature brings a Bitcoin analogue to ICP can be transferred at the speed of the ICP and with low transaction fees.
+coverImage: /img/how-it-works/direct-integration-with-bitcoin.600x300.jpg
+---
+
+![](/img/how-it-works/direct-integration-with-bitcoin.600x300.jpg)
+
+# Chain-key tokens
+
+Chain-key tokens — and Chain-Key Bitcoin (ckBTC) — are a cryptography-based replacement to wrapped tokens with strong decentralization advantages: Chain-key tokens eliminate the risks associated with the traditional intermediary-based token wrapping, while achieving the same desirable properties as wrapping. *Chain-key cryptography* makes this possible: Taking the example of Bitcoin, a canister smart contract can own ECDSA key pairs and derive Bitcoin addresses to which transfers of real Bitcoin can be made on the Bitcoin network. When receiving bitcoin, the canister mints and issues ckBTC in a 1:1 ratio to the sender of the bitcoin. Conversely, redeeming ckBTC for the underlying bitcoin removes the ckBTC from circulating supply and refunds the bitcoin. This makes a chain-key token a ‘twin’ on ICP of the original token, with high transaction speed and low fees.
```

Comment:
"... same desirable properties as wrapping."

The "as wrapping" part is superfluous. Wouldn't it be appropriate to state here what these properties are?
So, "... same desirable properties such as ..."?

Alternatively, we could just state the most important property:
"...while also having the same effect of making a token from another blockchain available for transfers and trading."

Reply:
"This makes a chain-key token a ‘twin’ on ICP of the original token" > "This makes a chain-key token a ‘twin’ of the original token with the same properties and valuation but hosted on the Internet Computer"